pyproject.toml

```diff
@@ -86,6 +86,7 @@ tests = [
 ]
 
 doc = [
+  "atsphinx-mini18n==0.4.1",
```

Comment:
I created https://github.com/ansys/pymapdl/pull/3967 because I found it inconvenient that this isn’t automatically formatted.